roles/power_aix_bootstrap/tasks/yum_install.yml

```diff
@@ -1,98 +1,105 @@
 # Copyright (c) IBM Corporation 2020
 ---
 # CHECK for Yum on inventory host
-  - name: Check for existence of yum
-    raw: "rpm -q --quiet yum && echo true || echo false"
-    register: yum_exists
+- name: Check for existence of yum
+  raw: "rpm -q --quiet yum && echo true || echo false"
+  register: yum_exists
+  changed_when: false
```

Comment:
Why this was added here?

Reply:
if you are asking on the `changed_when: false` then it is the same as the answer from the previous comment